doc/source/tutorial/stats/sampling_pinv.rst

```diff
@@ -0,0 +1,271 @@
+
+Polynomial interpolation based INVersion of CDF (PINV)
+======================================================
+
+.. currentmodule:: scipy.stats
+
+* Required: PDF
+* Optional: CDF, mode, center
+* Speed:
+
+  * Set-up: (very) slow
+  * Sampling: (very) fast
+
+Polynomial interpolation based INVersion of CDF (PINV) is an inversion method
+that only requires the density function to sample from a distribution. It is
+based on Polynomial interpolation of the PPF and Gauss-Lobatto integration
+of the PDF. It provides control over the interpolation error and integration
+error. Its primary purpose is to provide very fast sampling which is nearly
+the same for any given distribution at the cost of moderate to slow setup
+time. It is the fastest known inversion method for the fixed-parameter case.
+
+The inversion method is the simplest and most flexible to sample nonuniform
+random variates. For a target distribution with CDF :math:`F` and a uniform
+random variate :math:`U` sampled from :math:`\text{Uniform}(0, 1)`, a random
+variate X is generated by transforming the uniform random variate :math:`U`
+using the PPF (inverse CDF) of the distribution:
+
+.. math::
+    X = F^{-1}(U)
+
+This method is suitable for stochastic simulations because of its advantages.
+Some of the most attractive are:
+
+* It preserves the structural properties of the uniform random number sampler.
+* Transforms a uniform random variate :math:`U` one-to-one into non-uniform
+  random variates :math:`X`.
+* Easy and efficient sampling from truncated distributions.
+
+Unfortunately, the PPF is rarely available in closed form or too slow when
+available. For many distributions, the CDF is also not easy to obtain. This
+method addresses both the shortcomings. The user only has to provide the PDF
+and optionally a point near the mode (called "center") together with the size
+of the maximal acceptable error. It uses a combination of an adaptive and a
+simple Gauss-Lobatto quadrature to obtain the CDF and Newton's interpolation
+to obtain the PPF. The method is not exact, as it only produces random
+variates of the approximated distribution. Nevertheless, the maximal tolerated
+approximation error can be set close to the machine precision and its relative
+error is not much larger than :math:`10^{-15}`. The concept of u-error is used
+to measure and control the error. It is defined as:
+
+.. math::
+    \epsilon_{u}(u) = | u - F\left(F^{-1}_{a}(u)\right) |
+
+where :math:`u \in (0, 1)` is a quantile where we want to measure the error
+and :math:`F^{-1}_a` is the approximated PPF of the given distribution.
+
+The maximal u-error is the criterion for approximation errors when calculating
+the CDF and PPF numerically. The maximal tolerated u-error of an algorithm is
+called the u-resolution of the algorithm and denoted by :math:`\epsilon_{u}`:
+
+.. math::
+    \sup_{u \in (0,1)} | u - F\left(F^{-1}_{a}(u)\right) | \le \epsilon_{u}
+
+There are two drawbacks of using u-resolution:
```

Comment:
That sounds a bit negative :) I would say the following:

The main advantage of the u-error is that it can be easily computed if the CDF is available. We refer to [cite PINV paper] for a more detailed discussion.

Maybe we should even remove the drawbacks: it is probably not easy to understand why high/narrow peaks are a problem etc and we already explain that the method does not work well in that case

Reply:
> Maybe we should even remove the drawbacks: it is probably not easy to understand why high/narrow peaks are a problem etc and we already explain that the method does not work well in that case

Yup, this sounds good. I will add the advantage instead.

Reply:
Done.